src/csharp/Microsoft.Spark/Sql/RelationalGroupedDataset.cs

```diff
@@ -85,6 +90,25 @@ internal RelationalGroupedDataset(JvmObjectReference jvmObject, DataFrame dataFr
         public DataFrame Sum(params string[] colNames) =>
             new DataFrame((JvmObjectReference)_jvmObject.Invoke("sum", (object)colNames));
 
+        /// <summary>
+        /// Pivots a column of the current `DataFrame` and performs the specified aggregation.
+        /// </summary>
+        /// <param name="colName">Name of column to pivot on</param>
+        /// <returns>New DataFrame object with pivot applied</returns>
+        public RelationalGroupedDataset Pivot(string colName) =>
+            new RelationalGroupedDataset(
+                (JvmObjectReference)_jvmObject.Invoke("pivot", colName));
+
+        /// <summary>
+        /// Pivots a column of the current `DataFrame` and performs the specified aggregation.
+        /// </summary>
+        /// <param name="colName">Name of column to pivot on</param>
+        /// <param name="values">Name of distinct values to pivot</param>
+        /// <returns>New DataFrame object with pivot applied</returns>
+        public RelationalGroupedDataset Pivot(string colName, params string[] values) =>
```

Comment:
Let's use https://github.com/dotnet/spark/pull/598 for `Pivot`.

Reply:
Sure I'll remove `Pivot` from this PR.